### PR TITLE
fix: issue #3693 'save time with design patterns' font size and some …

### LINKED ISSUE #3693: [MeshMap] Issues with spacing in "Save time with patterns" section
A few padding / spacing issues in mobile view:

Screenshot 2023-02-03 at 3 32 53 PM

The line height / padding between lines is a bit too much in small to medium window sizes. Please reduce the whitespace.

Screenshot 2023-02-03 at 3 34 52 PM

_Originally posted by @leecalcote in https://github.com/layer5io/layer5/issues/3690#issuecomment-1416427185_

### PR DESCRIPTION
**Pull request by ritiksaxena124**
…layout changes

Signed-off-by: Ritik Saxena <ritiksaxena124@gmail.com>

**Description**
Changed some font-size and some display property which fixes the view on mobile and medium sized screens.

This PR fixes #3693 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 34908bee55acc8102a7104dd17be0575cf3c02ae at: https://63e647782630ff0bedb1367f--layer5.netlify.app

**Review by Shivam-AfA — COMMENTED** [review]
@ritiksaxena124 You have done a realyy good job for the mobile view. But you do not need to center align the text in medium and large resolutions (i.e before the transition to mobile view takes place). It should remain right aligned.

Instead of this,
image

We need this.
image

**Comment by ritiksaxena124** [comment]
Okay, I will make them aligned right again and make a PR soon.

**Comment by ritiksaxena124** [comment]
Please do have a look

Screenshot from 2023-02-10 23-42-38

**Comment by leecalcote** [comment]
Thank you, @ritiksaxena124. New deploy preview is building...

File: src/sections/Meshmap/meshmap-catalog.js [hunk 1]
```diff
@@ -31,69 +29,61 @@ margin: 7rem 0 7rem;
 min-height: fit-content;
 
 .patterns {
-  max-height: 30rem;
-  padding-bottom: 2rem;
-  margin-bottom: 3rem;
```
Comment (leecalcote) [inline src/sections/Meshmap/meshmap-catalog.js:36]: margin-bottom: 1rem;

File: src/sections/Meshmap/meshmap-catalog.js [hunk 2]
```diff
@@ -31,69 +29,61 @@ margin: 7rem 0 7rem;
...
+  padding-bottom: 5rem;
```
Comment (leecalcote) [inline src/sections/Meshmap/meshmap-catalog.js:32]:
```suggestion
  margin-bottom: 1rem;
  padding-bottom: 5rem;
```

**Comment by l5io** [comment]
🚀 Preview for commit c30b4e961984b47b454451951dd84dc2dd2ca5f8 at: https://63e6a51ca25c0f16e42a778c--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
> Please do have a look
> 
> Screenshot from 2023-02-10 23-42-38

Equal spacing on both side of the vertical line is desired. @ritiksaxena124

**Comment by ritiksaxena124** [comment]
Screenshot

Is this good ?

**Comment by Nikhil-Ladha** [comment]
> Screenshot
> 
> Is this good ?

Yes, please do push a fix for the same.

**Comment by l5io** [comment]
🚀 Preview for commit 516b2f2d3c6a8b81eae97b376410c16b088996d9 at: https://63e7c13bcc473a4b13202bab--layer5.netlify.app

**Pull request closed without merging**